### PR TITLE
Enable exp flag for MLflow conversion

### PR DESCRIPTION
**Pull request by novaturient95**

### CODE REVIEW

**Review by skasturi — APPROVED** [review]

File: assets/training/model_management/components/convert_model_to_mlflow/spec.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+version: 0.0.8
 type: command
 
 is_deterministic: True
 
 display_name: Convert Hugging face models to MLflow
 description: Component converts Huggingface models to MLflow model packaging format
 
-environment: azureml://registries/azureml/environments/model-management/versions/10
+environment: azureml://registries/azureml/environments/model-management/versions/11
```
Comment (skasturi) [inline assets/training/model_management/components/convert_model_to_mlflow/spec.yaml:12]: This needs to change as well after the new version gets pushed to PyPi isn't it?

Comment (novaturient95) [reply assets/training/model_management/components/convert_model_to_mlflow/spec.yaml:12]: yes, incremented the env version to make sure we pick the right env after release

**Review by skasturi — DISMISSED** [review]

**Comment by novaturient95** [comment]
Merge once model-management env version 11 is released to prod

File: assets/training/model_management/components/convert_model_to_mlflow/spec.yaml (hunk 1 above)
Comment (novaturient95) [inline assets/training/model_management/components/convert_model_to_mlflow/spec.yaml:12]: Merge once model-management env version 11 is released to prod

**Review by vikasagrawal-ms — DISMISSED** [review]

**Review by vikasagrawal-ms — APPROVED** [review]

**Review by skasturi — APPROVED** [review]

**Pull request merged**